bin/s2nd.c

```diff
@@ -311,12 +313,93 @@ void usage()
     fprintf(stderr, "    Accept client's TLS maximum fragment length extension request\n");
     fprintf(stderr, "  --ocsp\n");
     fprintf(stderr, "    Path to a DER formatted OCSP response for stapling\n");
+    fprintf(stderr, "  -s\n");
+    fprintf(stderr, "  --self-service-blinding\n");
+    fprintf(stderr, "    Don't introduce 10-30 second delays on TLS Handshake errors\n");
```

Comment:
I think we should add a comment that this option should not be used outside of test environments. Folks using `s2nd` as an example app might see this and assume it's a normal option to use.